msg/RadarPoint.msg

```diff
@@ -0,0 +1,10 @@
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+float32 range                            # Distance (m) from the sensor to the detected point.
+float32 angle                            # Angle (in radians) in the azimuth plane between the sensor and the detected point.
+                                         #   Negative angles are anticlockwise from the sensor.
```

Comment:
counter-clockwise?

Reply:
Here is the [ROS driver link](https://www.smartmicro.com/fileadmin/media/Downloads/Automotive_Radar/Software/umrr_driver.tar.xz.zip) for the UMRR radar from SmartMicro

Reply:
Thanks for the link to the UMRR Radar. I've added this to the analysis